project/Kang-Ding-Xin-NYCU/README.rst

```diff
@@ -0,0 +1,54 @@
+Basic Information
+=================
+
+PolygonAreaCalaulator
+
+`Kang-Ding-Xin-NYCU <https://github.com/Kang-Ding-Xin-NYCU>`__ for
+`PolygonAreaCalaulator <https://github.com/Kang-Ding-Xin-NYCU/PolygonAreaCaculator/README.rst>`__
+:https://github.com/Kang-Ding-Xin-NYCU/PolygonAreaCaculator
+
+Problem to Solve
+================
+
+* Easily get the the area of polygons and the image of polygons
+
+System Architecture
+===================
+
+* input : vertices of polygons
+* produces results : do some matrix operations and draw the image
+* result : the area of polygons and the image of polygons
+
+API Description
+===============
+code
+
+.. code:: c++
+
+    class polygon{
+        vector<vertices> verticeslist;
+        void area();
+        void draw();
+    };
+
+Engineering Infrastructure
+==========================
+
+* build : make
+* version control : git
+
+Schedule
+========
+
+* Week 1 (09/20):survey the method to implement the area calculation and drawing image
```

Comment:
Please include automatic testing and CI early in the schedule.  Think about how it improves your development productivity.